test/useRootCloseSpec.js

```diff
@@ -22,14 +22,17 @@ describe('RootCloseWrapper', () => {
   });
 
   describe('using default event', () => {
+    // eslint-disable-next-line mocha/no-setup-in-describe
```

Comment:
wait, this is actually wrong though?

Reply:
as in these are mis-written test cases that don't assert

Reply:
>as in these are mis-written test cases that don't assert

I agree, maybe we should add assertions or remove them?

Reply:
this PR was accidentally merged by Dependabot. @taion can you configure on this repo, that at least 2 reviewers are required?

Reply:
oh, wait, these do actually assert. we're fine, then. the rule says to disable it for dynamically generated test cases like this